conf/test.config

```diff
@@ -36,4 +38,11 @@ params {
 
     // STAR index (optional)
     star_index          = 'https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/genome/index/star/star.tar.gz'
+
+    // Annotation
+    snpeff_db           = 'GRCh38.99'
+    vep_genome          = 'GRCh38.p13'
+    vep_cache_version   = '104'
+    vep_species         = 'homo_sapiens'
```

Comment:
I'd recommend to test not with human as the db to download is huge.
Can you try if it annotates with C.elegans (even if variants are human).
It's what I do in Sarek

Reply:
oh ok. I saw that in Sarek. I can do that

Reply:
Added C.elegans!